docs_source_files/content/_index.fr.md

```diff
@@ -109,14 +108,16 @@ const {Builder, By, Key, until} = require('selenium-webdriver');
 
 (async function example() {
     let driver = await new Builder().forBrowser('firefox').build();
-    try {
-        await driver.get('https://www.google.com/ncr');
-        await driver.findElement(By.name('q')).sendKeys('cheese', Key.RETURN);
-        let firstResult = await driver.wait(until.elementLocated(By.css('h3>a')),10000);
-        console.log(await firstResult.getText());
-    } finally {
-        await driver.quit();
-    }
+
+    // Navigate to Url
```

Comment:
Oh, I think we went all the way to be without try/catch at all.
What I meant in the previous review is that the extra code added to handle the exceptions was not necessary for this basic examples.
But there was a try/catch that already existed before these changes, so please let's leave it.

Reply:
@diemol , do you want me to revert try/finally for js sample code ?

Reply:
@diemol . Updated